lib/membrane_rtc_engine/engine.ex

```diff
@@ -340,9 +341,12 @@ defmodule Membrane.RTC.Engine do
   def handle_init(options) do
     play(self())
 
+    trace_ctx = Membrane.RTC.Utils.create_otel_context("rtc: #{options[:id]}")
```

Comment:
Is the space in the name `rtc: ` intentional?

Reply:
There is no use for this space so it can be removed.